akka-http-tests/src/test/scala/akka/http/scaladsl/marshalling/MarshallingSpec.scala

```diff
@@ -106,31 +106,31 @@ class MarshallingSpec extends AnyFreeSpec with Matchers with BeforeAndAfterAll w
     "multipartMarshaller should correctly marshal multipart content with" - {
       "no parts" in {
         marshal(Multipart.General(`multipart/mixed`)) shouldEqual HttpEntity(
-          contentType = (`multipart/mixed` withBoundary randomBoundary).toContentType,
+          contentType = (`multipart/mixed` withBoundary randomBoundary()).toContentType,
```

Comment:
In MarshallingSpec, we override `multipartBoundaryRandom` to a fixed value, so `randomBoundary` actually returns a constant (which AFAICS this test relies on).

Wouldn't it be clearer to, instead of overriding `multipartBoundaryRandom`, just call `randomBoundary` once and use that value twice in the test?